bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/Strand.java

```diff
@@ -322,10 +336,12 @@ public WaitResult handleWaitAny(List<FutureValue> futures) throws Throwable {
         return waitResult;
     }
 
+    private BError createWaitOnSameFutureError() {
+        return ErrorCreator.createError(StringUtils.fromString("multiple waits on the same future is not allowed"));
```

Comment:
Shall we move this error to https://github.com/ballerina-platform/ballerina-lang/blob/master/bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties.